src/amo/pages/Addon/index.js

```diff
@@ -498,15 +498,10 @@ export class AddonBase extends React.Component {
           <Card className="Addon-header-info-card" photonStyle>
             {this.renderInstallError()}
 
-            {isFireFox && !isCompatible ? (
-              <AddonCompatibilityError
-                className="Addon-header-compatibility-error"
-                downloadUrl={compatibility.downloadUrl}
-                maxVersion={compatibility.maxVersion}
-                minVersion={compatibility.minVersion}
-                reason={compatibility.reason}
-              />
-            ) : null}
+            <AddonCompatibilityError
+              className="Addon-header-compatibility-error"
```

Comment:
I don't see `className` being a prop of the `AddonCompatibilityError` component. It should be declared as a prop and combined with `makeClassName()`.

Reply:
Interesting. I should have caught that. Looking at the code, I don't actually see that class being used anywhere, and it's certainly not doing anything currently because the code for `AddonCompatibilityError` never uses it. I'll just remove it from here.